pom.xml

```diff
@@ -75,7 +79,7 @@
 
     <properties>
         <checkstyle.skip>false</checkstyle.skip>
-        <java.min.version>1.7</java.min.version>
+        <java.min.version>1.8</java.min.version>
```

Comment:
I have intentionally held back the Java version because this library is more widely applicable than the Manta SDK.